backend/corpora/common/entities/entity.py

```diff
@@ -34,13 +41,23 @@ def _query(cls, key: typing.Union[str, typing.Tuple[str, str]]) -> typing.List[B
         raise NotImplementedError()
 
     @classmethod
-    def _load(cls, db_result: typing.List[Base]) -> "Entity":
+    def _load(cls, db_result: typing.List["Entity"]) -> typing.Union["Entity", None]:
         """
-        Parses a database query result into an Entity instance
-        :param db_result: list of query result rows
+        Parses query result rows produced by Entity._query into an instantiated Entity object.
```

Comment:
```suggestion

        Parses query result rows produced by Entity._query into an instantiated Entity object.
```
Just adding a line break for readability.